corehq/apps/hqwebapp/static/hqwebapp/js/bootstrap5/email-request.js

```diff
@@ -36,7 +36,7 @@ hqDefine('hqwebapp/js/bootstrap5/email-request', [
         self.$formElement.submit(() => {
             resetErrors();
 
-            const isDescriptionEmpty = !self.subjectText() && !self.descriptionText();
+            const isDescriptionEmpty = !self.subjectText();
```

Comment:
should the name be changed here as well? `isSubjectEmpty`.

Reply:
Ah, yes that should. This should match how we have it for the bootstrap 3 version.

Reply:
[d0ebdda](https://github.com/dimagi/commcare-hq/pull/34026/commits/d0ebddaffb18f44e2f954488a3abcb2fa6772563)